CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+<a name="7.0.0"></a>
+
+# [7.0.0](https://github.com/commercetools/ui-kit/compare/v6.1.0...v7.0.0) (2019-01-21)
+
+## BREAKING CHANGES
+
+- `Radio`: renamed to `RadioInput`
+- `Checkbox`: renamed to `CheckboxInput`
+- `Toggle`: renamed to `ToggleInput`
```

Comment:
Didn't also the `onChange` handler change, to now return an `event`?

Reply:
good catch 👍

Reply:
It was part of that PR and I didn't update the title 🤦‍♂️ 
I updated the changelog again